### PR TITLE
blog: post "The value of Urbit address space (1 of 3)"

### PR DESCRIPTION
**Pull request by urcades**
what it says on the tin

### CODE REVIEW

**Comment by urcades** [comment]
I'm fixing up an image still, ignore the header image material, focus on text stuff

File: content/blog/does-urbit-address-space-have-value-pt1.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+There’s a lot to discuss, so we’re breaking this discussion into three posts. The first explores what Urbit ID is, shares some history, and provides a few analogies that can help in understanding value on the network. The second looks at scarcity, utility, liquidity, and network effect. The third post explains lockups and availability.
+
+First, a brief refresher about what we’re doing. 
+
+The goal of Urbit is to deliver a complete, cohesive, and superior user experience for personal computing in a networked world. We’ve built a system that is simple, durable, and entirely owned by the user. Urbit gives autonomy back to users in a world where most of our computing and platforms are owned by centralized authorities. 
+
+Broadly speaking, Urbit is two pieces of technology designed to be used together: Urbit ID and[ Urbit OS](https://urbit.org/understanding-urbit/urbit-os/). This post deals primarily with Urbit ID, as that is the component of the Urbit system that is actively being traded on open markets.
```
Comment (matildepark) [inline content/blog/does-urbit-address-space-have-value-pt1.md]: The "Urbit OS" link has a leading space

File: content/blog/does-urbit-address-space-have-value-pt1.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+First, a brief refresher about what we’re doing. 
+
+The goal of Urbit is to deliver a complete, cohesive, and superior user experience for personal computing in a networked world. We’ve built a system that is simple, durable, and entirely owned by the user. Urbit gives autonomy back to users in a world where most of our computing and platforms are owned by centralized authorities. 
+
+Broadly speaking, Urbit is two pieces of technology designed to be used together: Urbit ID and[ Urbit OS](https://urbit.org/understanding-urbit/urbit-os/). This post deals primarily with Urbit ID, as that is the component of the Urbit system that is actively being traded on open markets. 
+
+We think of Urbit as a digital frontier with vast amounts of uncharted territory. Right now early pioneers are settling Urbit and creating communities and ecosystems that aren’t possible in the centralized version of the internet. Urbit IDs are the cornerstones of this activity.
+
+For a deeper dive into what Urbit is, check out our[ introductory materials](https://urbit.org/understanding-urbit/) or [Urbit for Normies](https://urbit.org/blog/urbit-for-normies/).
```
Comment (matildepark) [inline content/blog/does-urbit-address-space-have-value-pt1.md]: Likewise, leading space inside link text will be underlined on the site

File: content/blog/does-urbit-address-space-have-value-pt1.md (hunk 1 above)
Comment (urcades) [reply content/blog/does-urbit-address-space-have-value-pt1.md]: fixed

File: content/blog/does-urbit-address-space-have-value-pt1.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+In order to explore why Urbit IDs have value, we’ll start by explaining the basics of how they function.
+
+## Urbit ID basics ##
```
Comment (matildepark) [inline content/blog/does-urbit-address-space-have-value-pt1.md]: Remove second `##`

File: content/blog/does-urbit-address-space-have-value-pt1.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+So, fundamentally, Urbit ID is just a finite set of names and a system for assigning them. We’ve left out a few technical details here, but if you’re curious you can read our [introductory](https://urbit.org/understanding-urbit/urbit-id/) materials or, to go deeper, read the [docs](https://urbit.org/docs/) or the [source](https://github.com/urbit/azimuth).
+
+## Why we need a new address space
+
+In our view, the short history of the internet has been a lesson in centralization and consolidation. DNS is a cabal of root servers. IP is essentially controlled by a single entity. All of our naming schemes have been more or less centralized. Certificate signing is a mess of competing authorities. Search is Google. Hosting is AWS. And don’t even get us started on social networks.
+
+Centralization on its own isn’t bad, but it certainly has some negative implications, privacy being the most obvious. Beyond that the naming schemes we have on the internet today aren’t _personal;_they’re not controlled by their users.
```
Comment (matildepark) [inline content/blog/does-urbit-address-space-have-value-pt1.md]: needs a space after `personal;_`

File: content/blog/does-urbit-address-space-have-value-pt1.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+## Analogues and the intrinsic value of address space
+
+We’ve often said that Urbit address space, particularly stars and galaxies, have intrinsic value that should be understood beyond simply the price of the planets they contain. Another way of putting this is that, because stars and galaxies are infrastructure nodes, they’re uniquely positioned to generate cash flow from activities on the network beyond simply selling their planets. 
+
+Let’s dig a bit deeper into these activities through the lens of some analogues to the existing world. 
+
+DNS –
```
Comment (matildepark) [inline content/blog/does-urbit-address-space-have-value-pt1.md]: All these might be better as an h3 without `—`

Comment (urcades) [reply content/blog/does-urbit-address-space-have-value-pt1.md]: Good call, fixing

**Comment by urcades** [comment]
I've attached an image to this now, it's good to go on my end barring more editing

**Pull request merged**